### ISSUE TITLE
Output Images Compared to Examples

### ISSUE THREAD

**Issue by JoeLorenzoMontano** [llSourcell/Pokemon_GAN#9]
After completing the 5000 epoch run, my output image is nothing like yours... thoughts?!

![epoch4500](https://user-images.githubusercontent.com/6155201/35358070-4785f638-010a-11e8-803b-567086411720.jpg)

**Comment by denniswittich** [comment]
Same here. Even after 25.000 epochs ;)
Modified the code.. waiting for the results!

**Comment by JoeLorenzoMontano** [comment]
I played with the code a bit too but couldn't get it to output much better then the above image :/ Let me know how it goes for you!

**Comment by denniswittich** [comment]
What I realized is, that I had a problem with the alpha channel of the pngs. 
Add: 
``train_image = sess.run(image_batch)``
 ``save_images(train_image, [8, 8], newPoke_path + '/train.jpg')``
to line 255. This will show you the actual images used for training.
In my case there were lots of black rectangles in the images that came from the png conversion.
Just like those in my (and your) generated images.
(maybe the ``decode_jpeg`` method in line 40 should not be used to decode PNGs xD)
I went the easy way and converted all images to jpgs..
But still no final results yet.

**Comment by denniswittich** [comment]
![epoch21500](https://user-images.githubusercontent.com/23318166/36246809-c069355a-1231-11e8-95de-942e52e83fee.jpg)

with some small modifications I got this result (after ~20000 iters).
What I did is:

- convert all images to jpg (because of errors in decoding pngs with alpha)
- normalize examples and generated images to be in range [-1,1] (revert this before writing result images)
- in line 128 of pokeGan.py use bn1

I also tried to add batch normalization to the last layer of the generator, but that doesn't seem to work well. Still don't know why...

**Comment by aaronbjohnson** [comment]
@denniswittich I was glad to come across your solution to this issue. However, I'm very new to GANs and can't seem to keep my images from looking cloudy. I know it's an issue with the alpha channel, but I just can't seem to find how to make it work. I'm just not sure how to implement the changes you mentioned above. Any chance you have your code samples from these areas that I could reference? Thanks so much!

**Comment by b3nh** [comment]
@denniswittich got any tips on configuring the code to output higher resolution single frames? (not the 8x8 grid)

**Comment by teytaud** [comment]
Could someone release a successful checkpoint ?
